### DISCUSSION TITLE
Clear logos taking 2 seconds to populate.

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#808] [Discussion]
Is there any way I can get the clear logos to instantly populate as I have this issue when starting AH2 and is a bit strange.

The logo's start with normal fonts then a second or two later the clearlogo kicks in.

I have fanart, omdb and tmdb api keys in TMDB and using TMDB to get my widgets information. 

Am I doing something wrong or missing a step?
![20230209_195032](https://user-images.githubusercontent.com/97023369/217923702-cb8d2095-565a-488b-8ab1-d8a95a0306bc.jpg)

**Comment by jurialmunkey** [comment]
Enable the setting to get artwork for widgets.

**Reply by DROIDshield** [reply]
Did you mean these?
![20230210_012830](https://user-images.githubusercontent.com/97023369/217977360-7503fa3c-8086-4a7c-8ad4-8718b6ebe42a.jpg)

**Reply by jurialmunkey** [reply]
No in tmdbhelper settings.

If you aren't doing widget art lookups then you need to wait for the service to fill the value. Service takes a moment to start, so if the items don't have art by default then it will be blank. Enabling art lookups adds extra art directly to the item - it means slightly slower loading of widget but faster appearance of art after loaded.

**Reply by DROIDshield** [reply]
Thanks i did it and it's 99 percent better, the rest i think down to my 50gbs bloody Internet. Thanks again.
